pkg/controller/node/node_controller.go

```diff
@@ -379,13 +379,60 @@ func (ctrl *Controller) deleteMachineConfigPool(obj interface{}) {
 	// TODO(abhinavdahiya): handle deletes.
 }
 
+// Determine if masters are currently configured as schedulable
+func (ctrl *Controller) getMastersSchedulable() (bool, error) {
+	schedulerList, err := ctrl.schedulerList.List(labels.SelectorFromSet(nil))
+	if err != nil {
+		return false, fmt.Errorf("error while listing scheduler config %v", err)
+	}
+	for _, sched := range schedulerList {
+		if sched.Name == schedulerCRName {
+			return sched.Spec.MastersSchedulable, nil
+		}
+	}
+	return false, nil
+}
+
+// Determine if a given Node is a master
+func (ctrl *Controller) isMaster(node *corev1.Node) bool {
+	_, master := node.ObjectMeta.Labels[masterLabel]
+	return master
+}
+
+// Given a master Node, ensure it reflects the current mastersSchedulable setting
+func (ctrl *Controller) reconcileMaster(node *corev1.Node) {
```

Comment:
Can you call this function from https://github.com/openshift/machine-config-operator/blob/76e5737c1cfc183974c361d9593179535d97176a/pkg/controller/node/node_controller.go#L250, so that we avoid code duplication?

Reply:
done